src/posts/entities/post.repository.spec.ts

```diff
@@ -162,4 +184,17 @@ describe('PostRepository', () => {
       );
     });
   });
+
+  describe('deletPost', () => {
+    it('should fail if post doesnt exit', () => {
+      expect(postRepository.deletePost('nonexistent-uuid')).rejects.toThrow(
+        new NotFoundException('not-found-exception'),
+      );
+    });
+    it('should return deleted post, if post was found and deleted', () => {
```

Comment:
This test should be updated if you will follow my suggestion to remove the return from `deletePost` method in post.repository.ts